docker-nginx/templates/default.conf.template

```diff
@@ -28,10 +28,17 @@ log_format json-logger escape=json
 
 
 upstream django {
+  # Defines a shared memory zone between worker processes, necessary for resolve keyword
+  zone django 64k;
+
+  # Add Docker's DNS resolver with a short TTL and IPv6 turned off
+  resolver 127.0.0.11 valid=1s ipv6=off;
```

Comment:
Today we discussed a different issue with @suricactus which seems to be caused by ipv6. It is completely unrelated, so it would be interesting to know what the reason for disabling this here was.

Reply:
As far as I understand it, docker internal DNS is via IPv4 and so is there default networking. Would be interested though what problems were caused by ipv6. Generally if we don't use it, we disable it, since it might cause delays we don't want (most probably wouldn't be the case here).

Reply:
nginx also needs to be able to proxy resources from external services (S3), we had problems with a system where ipv6 was not enabled but a hostname was still resolved to an ipv6. Disabling ipv6 in nginx was one workaround that did the job.

Reply:
@toebivankenoebi I will show you tomorrow what it was about. Basically line 156:
```diff
    # required DNS
-    resolver 8.8.8.8;
+    resolver 8.8.8.8 ipv6=off;
```